dev-packages/browser-integration-tests/suites/tracing/browserTracingIntegration/pageload/test.ts

```diff
@@ -30,6 +30,7 @@ sentryTest('creates a pageload transaction with url as source', async ({ getLoca
     [SEMANTIC_ATTRIBUTE_SENTRY_SAMPLE_RATE]: 1,
     [SEMANTIC_ATTRIBUTE_SENTRY_SOURCE]: 'url',
     [SEMANTIC_ATTRIBUTE_SENTRY_OP]: 'pageload',
+    ['sentry.idle_span_finish_reason']: 'idleTimeout',
```

Comment:
L: Would it make sense to create a variable for this string as well?

Reply:
we have, but not publicly exported (not meant to be used by users) :)